File: symfony/bike/src/CategorieBundle/Entity/categorie.php
<?php

namespace CategorieBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
/**
 * categorie
 *
 * @ORM\Table(name="categorie")
 * @ORM\Entity(repositoryClass="CategorieBundle\Repository\categorieRepository")
 */
class categorie
{
    /**
     * @var integer
     *
     * @ORM\Column(name="ID_C", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idC;

    /**
     *
     * @var string
     * @Assert\NotBlank
     * @Assert\Length(min=3)
     * @ORM\Column(name="Lib_C", type="string", length=255, nullable=false)
     */
    private $libC;

    /**
     * @var string
     * @Assert\NotBlank
     * @Assert\Length(min=10)
     * @ORM\Column(name="Description", type="string", length=255, nullable=false)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="Type", type="string", length=255, nullable=false)
     */
    private $type;

    /**
     * @return int
     */
    public function getIdC()
    {
        return $this->idC;
    }

    /**
     * @param int $idC
     */
    public function setIdC($idC)
    {
        $this->idC = $idC;
    }

    /**
     * @return string
     */
    public function getLibC()
    {
        return $this->libC;
    }

    /**
     * @param string $libC
     */
    public function setLibC($libC)
    {
        $this->libC = $libC;
    }

    /**
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription($description)
    {
        $this->description = $description;
    }

    /**
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType($type)
    {
        $this->type = $type;
    }

}

File: bikeMobile/src/models/Categorie.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package models;



/**
 *
 * @author aymen
 */
public class Categorie {
    private int ID_C;
    private String Lib_C;
    private String Description;
    private String Type;

    public int getID_C() {
        return ID_C;
    }

    public Categorie(int ID_C, String Lib_C, String Description, String Type) {
        this.ID_C = ID_C;
        this.Lib_C = Lib_C;
        this.Description = Description;
        this.Type = Type;
    }

    public Categorie(String Lib_C, String Description, String Type) {
        this.Lib_C = Lib_C;
        this.Description = Description;
        this.Type = Type;
    }

    public String getLib_C() {
        return Lib_C;
    }

    public void setLib_C(String Lib_C) {
        this.Lib_C = Lib_C;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String Description) {
        this.Description = Description;
    }

    public String getType() {
        return Type;
    }

    public void setType(String Type) {
        this.Type = Type;
    }

  

   
    @Override
    public String toString() {
        return "Categorie{" + "Lib_C=" + Lib_C + ", Description=" + Description + ", Type=" + Type + '}';
    }
    
    
}
